support-frontend/assets/helpers/tracking/quantumMetric.ts

```diff
@@ -5,8 +5,9 @@ import { isSwitchOn } from 'helpers/globalsAndSwitches/globals';
 import type { IsoCurrency } from 'helpers/internationalisation/currency';
 import type { BillingPeriod } from 'helpers/productPrice/billingPeriods';
 import type { ProductPrice } from 'helpers/productPrice/productPrices';
+import type { SubscriptionProduct } from 'helpers/productPrice/subscriptions';
```

Comment:
This file is getting a little bigger, could potentially be worth adding some comment headers in to help break up the file slightly? It seems we have: `types`, `sending events`, and `initialisation`? Also, potentially within those sections I personally prefer the most important functions (i.e the ones being exported) right at the top, and then all the various helpers and shared functions that are just internal to the file after

Reply:
I've put some comment headers in for now

Reply:
nice - it's looking great!